src/style.css

```diff
@@ -1,3 +1,90 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.list__logo {
+  width: 40px;
+  height: 40px;
+  align-items: center;
+}
+
+.header__list {
+  margin-left: 50px;
+  margin-right: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  text-transform: uppercase;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  margin: 0;
+}
+
+.nav__item {
+  text-decoration: none;
+  list-style-type: none;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  display: block;
+  padding: 6px;
+  padding-top: 22px;
+  padding-bottom: 22px;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
+
+.is-active {
+  color: #00acdb;
+}
+
+.nav__link:hover {
+  color: #00acdb;
+  cursor: pointer;
+}
+
+.nav__item:first-child::after {
+  content: " ";
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdb;
+  display: block;
+  margin-left: 6px;
+}
```

Comment:
it must be for `is-active` class ❗  -> `is-active::after`
must have `position: absolute` with `bottom: 0`, `left: 0`, and `position: relative` for `.is-active`